docs/infra/merge_queue.md

```diff
@@ -16,19 +21,36 @@ With the merge queue, the button will now look like this:
 
 
 Unlike "Squash and merge", the new button does not immediately land the PR on the target branch, but instead enqueues your PR for extra checks in the merge queue. If you do not do anything, eventually your PR will land.
-Can I still use autosubmit?
-Yes! Feel free to continue using the autosubmit label as normal. The bots will run your PR through the merge queue and land it automatically.
-What's "Remove from queue"? Can I click it?
+
+---
+
+### Can I still use autosubmit?
+
+Yes! Feel free to continue using the `autosubmit` label as normal. The bots will run your PR through the merge queue and land it automatically.
+
+If you have an older PR from before this change, `autosubmit` may refuse to merge ot. Resync your pull request to resolve this issue.
+
+---
+
+### What's "Remove from queue"? Can I click it?
+
 While your PR is enqueued, Github will show the following button:
 
 ![Remove from queue button](https://github.com/flutter/assets-for-api-docs/blob/main/assets/wiki/remove_queue.png)
 
 When you press this button, GitHub will remove your PR from the queue, and the PR will no longer be landed by any automated machinery.
 
-WARNING: please only use this button in case of emergency. Adding and removing your PR from the queue costs in CI resources. Any CI work being done for your PR and all PRs that follow yours will be stopped and restarted.
-What if I have questions?
-If you have any questions or concerns, or If you suspect you are unable to land your PRs for reasons other than the familiar test failures and flakes, please contact any of: @yjbanov, @jtmcdole, @christopherfujino, @Piinks.
-Why do we need the merge queue?
-The main reason for this change is that flutter/engine and flutter/flutter repos will be combined into one "monorepo". In order to preserve the current lightweight dev cycle for the framework code, we need to provide pre-built engine artifacts. This way you don't need to compile any C++ code, or install extra tooling (e.g. depot_tools), when hacking on the framework alone. The merge queue is what will be building those engine binaries.
+> [!WARNING]
+> Please only use this button in case of emergency. Adding and removing your PR from the queue costs in CI resources. Any CI work being done for your PR and all PRs that follow yours will be stopped and restarted.
+
+---
+
+### What if I have questions?
+
+If you have any questions or concerns, or If you suspect you are unable to land your PRs for reasons other than the familiar test failures and flakes, please contact any of: [@yjbanov](https://github.com/yjbanov), [@jtmcdole](https://github.com/jtmcdole), [@christopherfujino](https://github.com/christopherfujino), [@Piinks](https://github.com/piinks).
```

Comment:
I don't think there's any messaging in github that would allow people to contact users directly. Should this point people to a discord chat room where the FAQ can be answered?

Also, since this document reads like a FAQ, I would move this "What if I have **other** questions?" to the bottom, so people read the whole document before resorting to direct contact.

Reply:
We have other docs that refer to Gtihub handles for contact. I think the expectation has been that they will ping on an issue, including if that issue is a question that needs answering. :)